ddl/reorg.go

```diff
@@ -350,26 +387,6 @@ func getReorgInfo(d *ddlCtx, t *meta.Meta, job *model.Job, tbl table.Table) (*re
 	if tbl == nil {
 		return info, errors.Trace(err)
 	}
-
-	// init the reorg meta info of job.
-	if job.ReorgMeta == nil {
```

Comment:
Can we make sure that the `job.ReorgMeta` is never nil when `SnapshotVer` is not 0?